tp-w8961n-status.py

```
@@ -10,18 +10,18 @@ def __init__(self, host):
 
     def __enter__(self):
         self.con = telnetlib.Telnet(self.host, 23, 5)
```

Ruff (0.13.1)

Telnet is considered insecure. Use SSH or some other encrypted protocol.

(S312)